examples/layer-browser/src/app.js

```diff
@@ -222,6 +228,7 @@ class App extends PureComponent {
         { this._renderMap() }
         { !MAPBOX_ACCESS_TOKEN && this._renderNoTokenWarning() }
         <div id="control-panel">
+          <button onClick={this._onQueryObjects}>Query Rendered Objects</button>
```

Comment:
"Query Objects" instead of "Query Rendered Objects"? Sounds like a Mapbox leftover...